### ISSUE TITLE
Remove ComponentReference

### ISSUE THREAD

**Issue by borismarin** [LEMS/jLEMS#54]
After discussions at UCL with @robertcannon, we have decided that ComponentReferences are just adding entropy to LEMS, since all the cases where they are used can be straightforwardly implemented using Child(ren)s. Relates to #32.

**Comment by borismarin** [comment]
With this decision in mind, it seems that all attributes in a `<Component>` definition should be correspond to parameter values, isn't that right @robertcannon ? That will greatly simplify our lives, as we can bind the "any" attributes to something else than String.

**Comment by robertcannon** [comment]
Yes, I think you're right, for the free ones. You've still got "type" and "extends" and possibly others with fixed names in future, but for normal components, the others would just set parameter values.
Though they won't necessarily all be physical quantities - the network stuff has xpath-like node selectors attribute values too.
